### DISCUSSION TITLE
Where is .env located?

### DISCUSSION THREAD

**Discussion by koayst** [FlowiseAI/Flowise#2141] [Q&A]
Hi,

I am using Windows 11 and I would like to enable debug mode.

From the documentation: [Flowise Documentation](https://docs.flowiseai.com/configuration/environment-variables) there is a .env file and I need to set DEBUG to TRUE.

Where is .env located ?

**Comment by toi500** [comment]
In the docker folder there is a template: https://github.com/FlowiseAI/Flowise/tree/main/docker

You can also read in there:

> You can specify the following variables in the .env file inside packages/server

**Reply by koayst** [reply]
-I didn't use Docker
-I downloaded NodeJS zip file and unzip it to "node-v21.7.2-win-x64" directory. Then change directory to that directory.  This is because it is a locked-up laptop (company policy)
-Open a command prompt and run "npm install -f flowise"
-Run flowise "npx flowise start"

I can actually start flowise like: "npx flowise start --DEBUG=true".  I just wonder if I would use .env file, where should .env file be placed.

Thanks.

**Comment by toi500** [comment]
Check the packages/server
https://github.com/FlowiseAI/Flowise/tree/main/packages/server

**Reply by sanjin1985** [reply]
I use "npm install -g flowise" , can‘t find  the packages/server，
<img width="508" alt="image" src="https://github.com/user-attachments/assets/173819cb-69ab-462b-b80e-f7f292eb8cde">
